Client/qtTeamTalk/mainwindow.cpp

```diff
@@ -922,7 +922,7 @@ void MainWindow::processTTMessage(const TTMessage& msg)
             //update AGC, denoise, etc. if changed
             updateAudioConfig();
 
-            update_ui = true;

```

Comment:
Why not just have one "slotUpdateUI()" at the bottom instead of calling it in every case-label? Also here the slotUpdateUI(); is called prior to invoking the event so actually the menu-items will not have been updated.